src/simulation/Simulation.cpp

```diff
@@ -3036,6 +3034,9 @@ int Simulation::get_normal_interp(int pt, float x0, float y0, float dx, float dy
 
 void Simulation::kill_part(int i)//kills particle number i
 {
+	if (i < 0 || i >= NPART)
+		return;
+	
```

Comment:
DELET

Reply:
check ur PMs jabob1

Reply:
ok, I'll look at the thing you sent me in more detail when I'm done playing animal crossing